test/screens/v2/widget_instance/cr_departures_test.exs

```diff
@@ -53,6 +53,86 @@ defmodule Screens.V2.WidgetInstance.CRDeparturesTest do
     end
   end
 
+  describe "serialize_departure/5" do
+    test "serializes a departure with a schedule and a prediction" do
+      now = ~U[2024-08-28 18:08:30.883227Z]
+
+      departure = %Departure{
+        schedule: %Schedule{
+          id: "schedule-1",
+          trip: %Trip{id: "trip-1", headsign: "Somewhere"},
+          arrival_time: DateTime.add(now, 5, :minute),
+          departure_time: DateTime.add(now, 7, :minute)
+        },
+        prediction: %Prediction{
+          id: "prediction-1",
+          trip: %Trip{id: "trip-1", headsign: "Somewhere"},
+          arrival_time: DateTime.add(now, 5, :minute),
+          departure_time: DateTime.add(now, 7, :minute)
+        }
+      }
+
+      assert %{
+               prediction_or_schedule_id: "prediction-1"
+             } =
+               CRDeparturesWidget.serialize_departure(
+                 departure,
+                 "Somewhere",
+                 %{},
+                 "place-smwhr",
+                 now
+               )
+    end
+
+    test "serializes a departure with only a schedule" do
+      now = ~U[2024-08-28 18:08:30.883227Z]
+
+      departure = %Departure{
+        schedule: %Schedule{
+          id: "schedule-1",
+          trip: %Trip{id: "trip-1", headsign: "Somewhere"},
+          arrival_time: DateTime.add(now, 5, :minute),
+          departure_time: DateTime.add(now, 7, :minute)
+        }
+      }
+
+      assert %{
+               prediction_or_schedule_id: "schedule-1"
+             } =
+               CRDeparturesWidget.serialize_departure(
+                 departure,
+                 "Somewhere",
+                 %{},
+                 "place-smwhr",
+                 now
+               )
+    end
+
+    test "serializes a departure with only a prediction" do
+      now = ~U[2024-08-28 18:08:30.883227Z]
+
+      departure = %Departure{
+        prediction: %Prediction{
+          id: "prediction-1",
+          trip: %Trip{id: "trip-1", headsign: "Somewhere"},
+          arrival_time: DateTime.add(now, 5, :minute),
+          departure_time: DateTime.add(now, 7, :minute)
+        }
+      }
+
+      assert %{
+               prediction_or_schedule_id: "prediction-1"
+             } =
+               CRDeparturesWidget.serialize_departure(
+                 departure,
+                 "Somewhere",
+                 %{},
+                 "place-smwhr",
+                 now
+               )
+    end
+  end
+
   # describe "serialize_times_with_crowding/2" do
```

Comment:
I'm curious why these are here 🤔 It's been here for 2 years so it feels safe to remove. Will leave it up to you.

Reply:
Easy enough: https://github.com/mbta/screens/pull/2170